src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -1862,6 +1862,13 @@ internal static void CheckNullableReferenceTypeAndScopedMismatchOnImplementingMe
                                 allowVariance: true,
                                 invokedAsExtensionMethod: false);
                         }
+
+                        if (implementingMethod.HasUnscopedRefAttributeOnMethodOrProperty())
```

Comment:
>implementingMethod [](http://example.com/codeflow?start=28&length=18)

It looks like we are going to report this error even for implicit implementations. Is this really what we want? Even for implementations that are in a base method?

Reply:
`[UnscopedRef]` attribute can only be applied to `struct` instance methods and properties, so base types can be ignored.